From 498078cae67e113f7ec4677ce0dd5ef839b41161 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?An=C3=ADbal=20Lim=C3=B3n?= <anibal.limon@linaro.org>
Date: Fri, 28 Dec 2018 18:09:09 -0600
Subject: [PATCH] CMakeLists.txt: No CURL
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Aníbal Limón <anibal.limon@linaro.org>
---
 CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 226724ba9..26e714fa4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -73,12 +73,6 @@ if (BERKELEY_DB_FOUND)
 endif()
 
 
-# apt-transport-https dependencies
-find_package(CURL REQUIRED)
-if (CURL_FOUND)
-  set(HAVE_CURL 1)
-endif()
-
 # (De)Compressor libraries
 find_package(ZLIB REQUIRED)
 if (ZLIB_FOUND)
-- 
2.19.2

